docs/api/apiv3/tags/baseline_comparisons.yml

```diff
@@ -0,0 +1,245 @@
+  ```
+
+  Each timestamp should be given as [ISO-8601](https://en.wikipedia.org/wiki/ISO_8601) string, either as absolute date and time with timezone, e.g. `"2022-01-01T00:00:00Z"`, or as relative timestamp utilizing the [ISO-8601-Duration](https://en.wikipedia.org/wiki/ISO_8601#Durations) format, e.g. `"P-1Y"`, which is composed of an initial `"P"` for "Period", and a duration. `"P-1Y"` is interpreted as the relative timestamp "1 year ago".
+
+  Several timestamps should be passed as comma-separated list of these ISO-8601 strings to the `timestamps` parameter.
+
+  The timestamps should be given in ascending temporal order, such that the first timestamp is the **baseline** timestamp, and the last timestamp is the **current** timestamp.
+
+  The information of the work packages at the different `timestamps` is returned in the `"_embedded"` section `"attributesByTimestamp"`.
+
+  ```json
+  // /api/v3/work_packages?timestamps=2022-01-01T00:00:00Z,PT0S
+  {
+    "_type": "WorkPackageCollection",
+    "total": 1,
+    "_embedded": {
+      "elements": [
+        {
+          "_type": "WorkPackage",
+          "id": 1528,
+          "subject": "Current subject of the work package",
+          // other attributes ...,
+          "_links": {
+            "self": {
+              "href": "/api/v3/work_packages/1528?timestamps=2022-01-01T00:00:00Z,PT0S"
+            }
+          },
+          "_meta": {
+            "matchesFilters": true,
+            "exists": true,
+            "timestamp": "PT0S"
+          },
+          "_embedded": {
+            "attributesByTimestamp": [
+              {
+                "subject": "Original subject of the work package",
+                "_meta": {
+                  "matchesFilters": true,
+                  "exists": true,
+                  "timestamp": "2022-01-01T00:00:00Z"
+                },
+                "_links": {
+                  "self": {
+                    "href": "/api/v3/work_packages/1528?timestamps=2022-01-01T00:00:00Z"
+                  }
+                },
+              },
+              {
+                "_meta": {
+                  "matchesFilters": true,
+                  "exists": true,
+                  "timestamp": "PT0S"
+                },
+                "_links": {
+                  "self": {
+                    "href": "/api/v3/work_packages/1528?timestamps=PT0S"
+                  }
+                }
+              }
+            ],
+          }
+        }
+      ]
+    },
+    "_links": {
+      "self": {
+        "href": "/api/v3/work_packages?timestamps=2022-01-01T00:00:00Z,PT0S"
+      }
+    }
+  }
+  ```
+
+  ## Filter Matching
+
+  The work-packages API supports filtering the query results by one or several search criteria. See: [Filters](../filters)
+
+  To find out whether a work package matches the given set of filter criteria, check `"matchesFilters"` in the `"_meta"` section for each timestamp:
```

Comment:
> matches the given set of filter criteria

I'd emphasis that it is matching it at that timestamp.

Reply:
➝ https://github.com/opf/openproject/pull/11783/commits/5a33ab64d384e89c6da23412661af7e4aea2182e